arbnode/batch_poster.go

```diff
@@ -1238,7 +1241,24 @@ func (b *BatchPoster) maybePostSequencerBatch(ctx context.Context) (bool, error)
 		b.building.msgCount++
 	}
 
-	if !forcePostBatch || !b.building.haveUsefulMessage {
+	var disablePosting bool
+	firstMsgTimeStamp := firstMsg.Message.Header.Timestamp
+	firstMsgBlockNumber := firstMsg.Message.Header.BlockNumber
+	batchNearL1BoundMinTimestamp := firstMsgTimeStamp >= l1BoundMinTimestamp && firstMsgTimeStamp <= l1BoundMinTimestamp+uint64(config.ReorgResistanceMargin/time.Second)
+	batchNearL1BoundMinBlockNumber := firstMsgBlockNumber >= l1BoundMinBlockNumber && firstMsgBlockNumber <= l1BoundMinBlockNumber+uint64(config.ReorgResistanceMargin/ethPosBlockTime)
```

Comment:
Sorry to keep adjusting what we're doing here, but we don't want to post a batch if it's out of the L1 bounds either, as that'd result in a reorg. So let's drop the first part of these conditionals.